src/rpdk/core/data_loaders.py

```diff
@@ -92,8 +92,8 @@ def get_file_base_uri(file):
 
 
 def _is_in(schema, key):
-    def contains(values):
-        return set(values).issubset(set(schema.get(key, [])))
+    def contains(value):
+        return value in schema.get(key, [])
 
     return contains
```

Comment:
seems like a weird layer of misdirection to have this function return a function rather than just having a function that takes all 3 parameters

Reply:
this is closure, i'd keep it in case we need to reuse or enhance the logic

Reply:
just seems unnecessary when the check is a one-liner with or without that function